features/org.eclipse.smarthome.feature.test/feature.xml

```diff
@@ -45,4 +45,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```

Comment:
Do you really want to have the sample handler factories in the test feature? I pulls dependencies to the automation component to the basic test feature.